test/jdk/java/util/concurrent/tck/ThreadTest.java

```diff
@@ -76,7 +76,6 @@ public void testGetAndSetUncaughtExceptionHandler() {
      * setDefaultUncaughtExceptionHandler.
      */
     public void testGetAndSetDefaultUncaughtExceptionHandler() {
-        assertNull(Thread.getDefaultUncaughtExceptionHandler());
```

Comment:
I think this is a very broad change in this test case and either shouldn't be done or should be done conditionally (I can't think of the right condition now because I haven't fully grasped the context of this PR).